src/systems/userinput/devices/daydream-controller.js

```diff
@@ -10,6 +13,15 @@ export class DaydreamControllerDevice {
     this.rayObjectRotation = new THREE.Quaternion();
     this.selector = `#player-${gamepad.hand}-controller`;
     this.pose = new Pose();
+    this.sittingToStandingMatrix = new THREE.Matrix4().makeTranslation(0, 1.6, 0);
+    navigator.getVRDisplays().then(d => {
```

Comment:
We could technically use
`scene.renderer.vr.getStandingMatrix()` here instead since its already a Matrix4 version of this, but I think decoupling from `WebVRManager` makes sense. 

We should add guards for `navigator.getVRDisplays` and `stageParameters` existing though. 

Just a note from checking the spec, it seems like this should not change out from under us unless we call `resetPose` so doing this only once here seems correct.

Reply:
I added a guard for `stageParameters` (and in fact this is `undefined` in chrome on android with daydream, so that's good). Why guard against `getVRDisplays`?